### ISSUE TITLE
Cannot enhance entity class with kotlin v.1.7.0

### ISSUE THREAD

**Issue by kklongming** [ebean-orm-tools/ebean-gradle-plugin#33]
KotlinCompile is no longer a subclass of AbstractCompile since kotlin v.1.7.0

plugin ver: 13.6.4

io/ebean/gradle/EnhancePlugin.groovy:112

**Comment by rbygrave** [comment]
So we need to determine how to detect KotlinCompile in a new way.

Note that we have https://github.com/ebean-orm/examples/tree/master/basic-gradle-kotlin ... as our basic gradle + kotlin project so this is the example we want to keep up to date and can submit PR's for this when things no longer work etc.

**Comment by rbygrave** [comment]
Notes on using a locally installed version of the plugin from localMaven:

In settings.gradle add:
```
pluginManagement {
  repositories {
    mavenLocal()
    gradlePluginPortal()
  }
}
```

In the build.gradle add:
```
buildscript{
  repositories {
    mavenLocal()
    dependencies{
      classpath 'io.ebean:ebean-gradle-plugin:13.6.4.1'
    }
  }
}
```
and in `plugins` remove the use of `id(io.ebean) ... ` there and add:

```
plugins {
  id('idea')
//  id('ebean') version '13.6.4.1'
  id('org.jetbrains.kotlin.jvm') version '1.7.0'
  id('org.jetbrains.kotlin.kapt') version '1.7.0'
}

apply plugin: 'io.ebean'
```

**Comment by rbygrave** [comment]
Fixed in version **`13.6.4.1`** which has been released to gradle plugin repository:

```
plugins {
  id('idea')
  id('io.ebean') version '13.6.4.1'
  id('org.jetbrains.kotlin.jvm') version '1.7.0'
  id('org.jetbrains.kotlin.kapt') version '1.7.0'
}
```
